src/TurbulenceClosures/isopycnal_rotation_tensor_components.jl

```diff
@@ -52,10 +52,11 @@ struct SmallSlopeIsopycnalTensor{FT} <: AbstractIsopycnalTensor
     minimum_bz :: FT
 end
 
-SmallSlopeIsopycnalTensor(; minimum_bz = 0) = SmallSlopeIsopycnalTensor(minimum_bz)
+SmallSlopeIsopycnalTensor(FT::DataType=Float64; minimum_bz = FT(0)) = SmallSlopeIsopycnalTensor(minimum_bz)
 
-@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers, slope_model::SmallSlopeIsopycnalTensor) where FT
+@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers, slope_model::SmallSlopeIsopycnalTensor, slope_limiter) where FT
     bx = ∂x_b(i, j, k, grid, buoyancy, tracers)
+    by = ℑxyᶠᶜᵃ(i, j, k, grid, ∂yᶜᶠᶜ, buoyancy_perturbation, buoyancy.model, tracers)
```

Comment:
why we need this here?

Reply:
to calculate the tapering factor, we can probably do it more efficiently, I have't worked out the details completely yet

Reply:
but this function never uses it

Reply:
I believe you missed the tapering in this function.

Reply:
I believe you missed the tapering in this function...

Reply:
Ah yeah! I was using #2603 😅, sure

Reply:
This comment above is about `bx` I presume, since `by` and `bz` are both used?

Reply:
Now they are after [943c782](https://github.com/CliMA/Oceananigans.jl/pull/2664/commits/943c782e1c9d4a62e91cb5d2d0a96723fbaf3b92)